conf/application.conf

```diff
@@ -209,6 +209,11 @@ microservice {
       port = 9050
       custom-payment-id = "1234567890"
     }
+    amls-art-market-participant-frontend {
+      host = localhost
+      port = 9223
+      url = "http://localhost:9223/amls-art-market-participant-frontend"
```

Comment:
do we need here whole url if there's host and port defined? baseUrl("amls-art-market-participant-frontend") should return first part of url: http://localhost:9223

Reply:
Will look into it.